m2cgen/assemblers/linear.py

```diff
@@ -70,6 +75,111 @@ def _get_coef(self):
         return self.model.params[:self.model.k_exog]
 
 
+class StatsmodelsGLMModelAssembler(StatsmodelsLinearModelAssembler):
+
+    def _final_transform(self, ast_to_transform):
+        link_function = type(self.model.model.family.link).__name__
+        link_function_lower = link_function.lower()
+        supported_inversed_functions = {
+            "logit": self._logit_inversed,
+            "power": self._power_inversed,
+            "inverse_power": self._inverse_power_inversed,
+            "sqrt": self._sqrt_inversed,
+            "inverse_squared": self._inverse_squared_inversed,
+            "identity": self._identity_inversed,
+            "log": self._log_inversed,
+            "cloglog": self._cloglog_inversed,
+            "negativebinomial": self._negativebinomial_inversed,
+            "nbinom": self._negativebinomial_inversed
+        }
+        if link_function_lower not in supported_inversed_functions:
+            raise ValueError(
+                "Unsupported link function '{}'".format(link_function))
+        fun = supported_inversed_functions[link_function_lower]
+        return fun(ast_to_transform)
+
+    def _logit_inversed(self, ast_to_transform):
+        return utils.div(
+            ast.NumVal(1.0),
+            utils.add(
+                ast.NumVal(1.0),
+                ast.ExpExpr(
+                    utils.sub(
+                        ast.NumVal(0.0),
+                        ast_to_transform))))
+
+    def _power_inversed(self, ast_to_transform):
+        power = self.model.model.family.link.power
+        if power == 1:
+            return self._identity_inversed(ast_to_transform)
+        elif power == -1:
+            return self._inverse_power_inversed(ast_to_transform)
+        elif power == 2:
+            return ast.SqrtExpr(ast_to_transform)
+        elif power == -2:
+            return self._inverse_squared_inversed(ast_to_transform)
+        elif power < 0:  # some languages may not support negative exponent
+            return utils.div(
+                ast.NumVal(1.0),
+                ast.PowExpr(ast_to_transform, ast.NumVal(1 / -power)))
+        else:
+            return ast.PowExpr(ast_to_transform, ast.NumVal(1 / power))
+
+    def _inverse_power_inversed(self, ast_to_transform):
```

Comment:
[Nitpick] That's a lot of inversions going on in a single place :) Furthermore they appear to mean somewhat different inversions (negative vs fractional exponent). I don't have any good suggestions here but perhaps you do.

Reply:
Yeah, I don't really like the current names too. The naming schema I used is `_` + `original_name` + `_inversed`. You are absolutely right that `inverse` prefix in original name means "negative sign of argument" while `inversed` postfix corresponds to "opposite" function (e.g. `log <-> exp`). I took `inversed` because we're actually implementing `inverse()` method here of original class.

Wikipedia suggests `anti` suffix: https://en.wikipedia.org/wiki/Inverse_function. But honestly, I don't like that. Maybe `reversed`? To distinguish from original `inverse`. But I guess it is not commonly used.

Reply:
I don't mind using anti here, but regardless of our choice the meaning will still be somewhat obscure. In any way I don't think that this decision should hold up this PR. I suggest we merge it and decide later. What do you think?

Reply:
Sure! Agree with your plan. 🙂